libs/community/pyproject.toml

```diff
@@ -101,7 +101,7 @@ types-requests = "^2.31.0.10"
 types-pillow = "^10.1.0.2"
 types-google-cloud-ndb = "^2.2.0.1"
 
-[tool.ruff]
+[tool.lint.ruff]
```

Comment:
should it be `tool.ruff.lint` too, or am I missing anything?

Reply:
Sorry, I updated it.